src/status_im/constants.cljs

```diff
@@ -237,12 +237,19 @@
 
 (def ^:const status-create-address "status_createaddress")
 
-(def ^:const path-root "m/44'/60'/0'/0")
-(def ^:const path-default-wallet "m/44'/60'/0'/0/0")
-(def ^:const path-whisper "m/43'/60'/1581'/0'/0")
+; BIP44 Wallet Root Key, the extended key from which any wallet can be derived
+(def ^:const path-wallet-root "m/44'/60'/0'/0")
+; EIP1581 Root Key, the extended key from which any whisper key/encryption key can be derived
+(def ^:const path-eip1581 "m/43'/60'/1581'")
+; BIP44-0 Wallet key, the default wallet key
+(def ^:const path-default-wallet (str path-wallet-root "/0"))
```

Comment:
before "m/44'/60'/0'/0/0" after "m/44'/60'/0'/0"/0" is it the same path ?

Reply:
for some reason before it was "0/"

Reply:
was it mistake?

Reply:
wait, not sure i understand, path looks the same to me:

before:

m/44'/60'/0'/0
m/44'/60'/0'/0/0

after:

m/44'/60'/0'/0
(str "m/44'/60'/0'/0" "/0")

looks the same to me, or i am missing something?

Reply:
oh my, confused by "